### ISSUE TITLE
completion of `-c` and `-l` should not appear after the first argument in `lb-run` and `lb-dev`

### ISSUE THREAD

**Issue by pseyfert** [pseyfert/lb-zsh#11]
Unfortunately, only `lb-run -l Brunel` but not `lb-run Brunel -l` will work, the tab completion accepts `-c` and `-l` at any position.

**Comment by pseyfert** [comment]
This is related to [LBCORE-718](https://its.cern.ch/jira/browse/LBCORE-718)

**Comment by pseyfert** [comment]
actually, since now commands get completed at the end of lb-run, it's not like hammering tab would automatically include the options at the end. Lowers the severity.

**Comment by pseyfert** [comment]
could be improved with

```
-_arguments -n '-l[list versions]' '-c[specify platform]:CMTCONFIG:__listcmt' '*: :__check_nightlies'
+_arguments -n '-l[list versions]' '-c[specify platform]:CMTCONFIG:__listcmt' '(- 1)*: :__check_nightlies'
```

(need to check for nightlies and -l -c combinations)

**Comment by pseyfert** [comment]
somewhat fixed'ish in 0467f30368ee332c0686f65a2b6f2dee4cc42cc4

`lb-run DaVinci/v42r2 -<TAB>` does not suggest `-l` or `-c`

but it's not supported to do `lb-run --nighties lhcb-head -l DaVinci` (sorry for that)
